### PR TITLE
WIP:Adding an Array

### PR DESCRIPTION
**Pull request by tobiKad**

### CODE REVIEW

File: eventRequirement.js [hunk 1]
```diff
@@ -634,6 +652,10 @@ OperandVariable.prototype.setPointers = function (entitiesArr) {
         this.operandValueOrObject().left.setPointers(entitiesArr);
         this.operandValueOrObject().right.setPointers(entitiesArr);
     }
+    if (this.operandType() == "arrayvalue") {
```
Comment (hfinger) [inline eventRequirement.js]: please reuse the arithmetic case. For example:
if (this.operandType() == "arithmetic" || this.operandType() == "arrayvalue")

File: eventRequirement.js [hunk 2]
```diff
@@ -661,6 +683,10 @@ OperandVariable.prototype.reAddEntities = function (entitiesArr) {
         this.operandValueOrObject().left.reAddEntities(entitiesArr);
         this.operandValueOrObject().right.reAddEntities(entitiesArr);
     }
+    if (this.operandType() == "arrayvalue") {
```
Comment (hfinger) [inline eventRequirement.js]: same like above. reuse arithmetic

**Review by hfinger — CHANGES_REQUESTED** [review]

File: eventRequirement.js (hunk 1 above)
Comment (tobiKad) [reply eventRequirement.js]: Done

File: eventRequirement.js (hunk 2 above)
Comment (tobiKad) [reply eventRequirement.js]: Done

**Pull request closed without merging**